assets/benchmarks/shaders/Benchmark_VsAluBound.hlsl

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "Benchmark.hlsli"
+
+VSOutput vsmain(
+    float4 position : POSITION,
+    float2 uv       : TEXCOORD,
+    float3 normal   : NORMAL,
+    float3 tangent   : TANGENT)
+{
+  VSOutput result;
+
+  const float4x4 jointMat[2] = {
+      1, 0, 0, 0,
+      0, 1, 0, 0,
+      0, 0, 1, 0,
+      0, 0, 0, 1,
+      1, 0, 0, 0,
+      0, 1, 0, 0,
+      0, 0, 1, 0,
+      0, 0, 0, 1,
+  };
+
+  float4 jointWeights = {
+    frac(uv.x), 1 - frac(uv.x), 0, 0
+  };
+
+  float4 jointIndices = {0, 1, 0, 0};
+
+  float4x4 skinMat = jointWeights.x * jointMat[int(jointIndices.x)] +
```

Comment:
the compiler probably is most likely able to optimize the case where the weight is 0 (in this case, we probably need to make the shader more complex to make sure the compiler is not doing any unwanted optimizations). can you put some comments here to check this later?